qa/evil-tests/src/test/java/org/elasticsearch/common/logging/EvilLoggerTests.java

```diff
@@ -340,26 +340,22 @@ public void testPrefixLoggerMarkersCanBeCollected() throws IOException, UserExce
     }
 
     public void testProperties() throws IOException, UserException {
-        final Settings.Builder builder = Settings.builder().put("cluster.name", randomAlphaOfLength(16));
-        if (randomBoolean()) {
-            builder.put("node.name", randomAlphaOfLength(16));
-        }
-        final Settings settings = builder.build();
+        final Settings settings = Settings.builder()
+                .put("cluster.name", randomAlphaOfLength(16))
+                .put("node.name", randomAlphaOfLength(16))
```

Comment:
`node.name` is always available at this point in the course of normal operation.